sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncClientEncryptionKey.java

```diff
@@ -110,6 +110,8 @@ String getLink() {
         return builder.toString();
     }
 
+    public static void doNothingButEnsureLoadingClass() {}
```

Comment:
For all of these public static APIs, we should have some java documentation, even if its a dummy one. Otherwise the API review will block it.

Reply:
Fixed (next iteration)